plugins/filter_aws/aws.c

```diff
@@ -434,7 +627,46 @@ static struct flb_config_map config_map[] = {
      " will be used: 'v1' or 'v2'. 'v2' may not work"
      " if you run Fluent Bit in a container."
     },
-
+    {
+     FLB_CONFIG_MAP_BOOL, "az", "false",
+     0, FLB_TRUE, offsetof(struct flb_filter_aws, availability_zone_include),
+     "Enable EC2 instance availability zone"
+    },
+    {
+     FLB_CONFIG_MAP_BOOL, "instance_id", "false",
+     0, FLB_TRUE, offsetof(struct flb_filter_aws, instance_id_include),
```

Comment:
These should default to "true".

A user can print the config map as help text using:

```
./fluent-bit -f aws -h
```

Reply:
Make sense, changed the default to "true".